alert-policies/apache-traffic-server/FreeCacheRAM.yml

```diff
@@ -0,0 +1,40 @@
+# Name of the alert
+name: Free Cache RAM Percent
+
+# Description and details
+description: |+
+  This alert is triggered if the cache RAM free lower than 5% for 5 minutes.
+# Type of alert
+type: STATIC
+
+# NRQL query
+nrql:
+
+  query: "SELECT (latest(global.proxy.process.cache.ram_cache.total_bytes) - latest(global.proxy.process.cache.ram_cache.bytes_used)) / latest(global.proxy.process.cache.ram_cache.total_bytes) * 100 FROM ATSCacheSampleMetrics"
+
+# Function used to aggregate the NRQL query value(s) for comparison to the terms.threshold (Default: SINGLE_VALUE)
+valueFunction: SINGLE_VALUE
+
+# List of Critical and Warning thresholds for the condition
+terms:
+  - priority: CRITICAL
+    # Operator used to compare against the threshold.
+    operator: BELOW
+    # Value that triggers a violation
+    threshold: 5
```

Comment:
```suggestion
    threshold: 10
```

Reply:
Hi @mdumpati,
As per your suggestions, I have made changes. Kindly review it.

Thanks.